mix.exs

```diff
@@ -72,6 +72,7 @@ defmodule Hummingbird.MixProject do
       {:plug, "~> 1.7"},
       {:elixir_uuid, "~> 1.2"},
       {:opencensus_honeycomb, "~> 0.3"},
+      {:jason, "~> 1.2"},
```

Comment:
```suggestion
      {:jason, "~> 1.0"},
```

yada yada as wide as possible to make it easy for consumers

Reply:
one of these days I'll get this right on my own.
when I do, could I get a log rip please?